src/schema/linkml/bertron_schema.yaml

```diff
@@ -162,7 +162,7 @@ classes:
         broad_mappings:
           - MIXS:0000009
         range: decimal degree
-        required: true
+        required: false
```

Comment:
I would leave these in as otherwise people can enter an empty or incomplete lat/long as a coordinates object.

Reply:
ok so you're saying leave latitude and longitude required within coordinates.. but make coordinates optional?